src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,73 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { getMovie } from '../../api';
+
+import { Movie } from '../../types/Movie';
+
+import { MovieCard } from '../MovieCard';
+
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  onAdd: (finded: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ onAdd }) => {
+  const [inputTitle, setInputTitle] = useState<string>('');
```

Comment:
Type will string by default
```suggestion
  const [inputTitle, setInputTitle] = useState('');
```